apps/ios/Shared/Views/Chat/ComposeMessage/ComposeView.swift

```diff
@@ -377,6 +377,7 @@ struct ComposeView: View {
                 }
             }
         }
+        .background(Color(uiColor: .tertiarySystemGroupedBackground))
```

Comment:
compose context/attachment rows look quite bad with this change (worse than just on background)